bus_presence_override/static/src/less/systray.less

```diff
@@ -0,0 +1,47 @@
+/* Copyright 2017 LasLabs Inc.
+   License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
+
+#userStatus {
+    &:hover i {
+        color: #d3d3d3 !important;
```

Comment:
`!important` is a dirty hack that should never be used, except without a damned good explanation. I assume there isn't one here though.